### PR TITLE
Release 0.3.x

### PR DESCRIPTION
**Pull request by langchain-infra**

### CODE REVIEW

File: charts/langsmith/README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # langsmith
 
-![Version: 0.2.11](https://img.shields.io/badge/Version-0.2.11-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+![Version: 0.3.1](https://img.shields.io/badge/Version-0.3.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
 
 Helm chart to deploy the langsmith application and all services it depends on.
 
+## Migrating from LangSmith 0.2.0 to 0.3.0
+
+LangSmith 0.3.0 should be a drop-in replacement for LangSmith 0.2.0. You can follow the generic upgrade instructions [here](https://github.com/langchain-ai/helm/blob/main/charts/langsmith/docs/UPGRADE.md).
```
[the remark this answers is not in the document]
Comment (barberscott) [reply charts/langsmith/README.md:9]: Mostly a nice to have to make reviewing these easier since the link should work in review.

**Review by barberscott — APPROVED** [review]
Minor nits.  Otherwise approved overall.  Will collate release notes from last few weeks and get a Doc update submitted.

File: charts/langsmith/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (langchain-infra) [reply charts/langsmith/README.md]: fixed

File: charts/langsmith/README.md.gotmpl [hunk 2]
```diff
@@ -4,6 +4,11 @@
 
 {{ template "chart.description" . }}
 
+## Migrating from LangSmith 0.2.0 to 0.3.0
+
+LangSmith 0.3.0 should be a drop-in replacement for LangSmith 0.2.0. You can follow the generic upgrade instructions [here](https://github.com/langchain-ai/helm/blob/main/charts/langsmith/docs/UPGRADE.md).
+Note: If you want to preserve old runs from 0.1.x, you will need to first upgrade to 0.2.0, migrate your runs, and then upgrade to 0.3.0.
```
[the remark this answers is not in the document]
Comment (langchain-infra) [reply charts/langsmith/README.md.gotmpl]: fixed

**Pull request merged**